src/plugins/felixhayashi/tiddlymap/js/graph/ViewAbstraction.js

```diff
@@ -81,23 +78,17 @@ class ViewAbstraction {
   }
 
   /**
-   * Gives the view a chance to rebuild its properties cache.
+   * This does nothing, but its return value specifies whether the view
+   * is expected to be different given the set of updates.
    *
    * @param {Updates} updates
-   * @return {boolean} True if changes affect parts of the view.
+   * @return {boolean} True if changes would affect parts of the view.
    */
   update(updates) {
 
     const { changedTiddlers } = updates;
 
-    if (updates[env.path.edgeTypes] || utils.hasKeyWithPrefix(changedTiddlers, this.getRoot())) {
-
-      this._rebuildCache();
-
-      return true;
-    }
-
-    return false;
+    return (updates[env.path.edgeTypes] || utils.hasKeyWithPrefix(changedTiddlers, this.getRoot()));
 
   }
 
```

Comment:
I considered changing the name of `update` to `updated` or `modified` or something, since it's now a zero-side-effect method. I think there might be some simplification possible in MapWidget now, where this is called.

Reply:
I understand your intention. In this case I would rename it to `hasUpdated()`. On the other hand, leaving it as a function with side effects also leaves the chance for us/others to add side effects in the future. This is why I called it `update` in analogy to the widget's `update` method. What would you suggest?

Reply:
If MapWidget is the only place where it's called, I'd recommend changing it. It can always be changed back later, since it's only two lines of code changed.

It _is_ only used in MapWidget, right? I couldn't find it anywhere else.

Reply:
> It is only used in MapWidget, right? I couldn't find it anywhere else.

yes

> If MapWidget is the only place where it's called, I'd recommend changing it. It can always be changed back later, since it's only two lines of code changed.

ok!